index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset='utf-8'>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@700&family=Poppins:ital,wght@0,400;0,700;1,400;1,700&family=Roboto:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link rel='stylesheet' type='text/css' media='screen' href='css/main.css'>
+</head>
+
+<body>
+    <nav class="row nav-bar">
+        <div class="cell title">Juan Luis Palacios Perez</div>
+        <div id="menu-btn" class="cell">
+            <img src="imgs/Icon%20-%20Menu.svg" alt="menu" />
+        </div>
+    </nav>
+    <main>
+        <section class="row blk-sec">
+            <div class="fl-box">
+                <h1 >Hey There.
+                    I&#039;m Juan
+                </h1>
+                <p class="p-sub">and I&#039;m a developer
+                </p>
+                <p id="p" class="paragraph">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                    tempor incididunt ut labore et dolore magna aliqua
+                </p>
+
+                <div class="cell">
+                    <a class="social-link">
+                        <img src="imgs/Icon%20-GitHub.svg" alt="GitHub icon" />
+                    </a>
+                    <a class="social-link">
+                        <img src="imgs/Icon%20-%20Linkedin.svg" alt="Linkedin icon" />
+                    </a>
+                    <a class="social-link">
+                        <img src="imgs/Icon%20-%20Angelist.svg" alt="Angelist icon" />
+                    </a>
+                    <a class="social-link">
+                        <img src="imgs/Icon%20-%20Twitter.svg" alt="Twitter icon" />
+                    </a>
+                    <a class="social-link">
+                        <img src="imgs/Icon%20-%20Medium.svg" alt="Medium icon" />
+                    </a>
+                </div>
```

Comment:
- [ ] Kindly use `ul & li` tag here for listing all social icons. for example 👇 
`<ul>`
 `<li><a href="#"><i class="fa fa-github"></i></a></li>`
`</ul>`
Requirements:-
- The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)